locales/ca/messages.json

```diff
@@ -1337,7 +1350,8 @@
         "message": "La declinació magnètica és l'angle entre el nord magnètic i el nord veritable. És diferent per a cada lloc de la terra. Podeu trobar la declinació de la vostra ubicació a Internet"
     },
     "configurationRangefinder": {
-        "message": "$t(initialSetupSensorSonar.message)"
+        "message": "$t(initialSetupSensorSonar.message)",
+        "description": "Don't translate!!!"
```

Comment:
🤦‍♂️

Reply:
Guess this is how crowdin works

Reply:
I mean the key should be present under the language it is used.